src/main/java/com/openshift/internal/restclient/DefaultClient.java

```diff
@@ -212,6 +207,18 @@ public String getResourceURI(IResource resource) {
 		}
 	}
 	
+	private RequestBody getPayload(String method, IResource payload) {
```

Comment:
cc @scabanovich I would be interested in your thoughts regarding my previous comment